lib/routes/surveyAdmin.js

```diff
@@ -0,0 +1,38 @@
+Router.route('/survey/admin', {
+  name: 'survey.admin',
+
+  waitOn() {
+    return [Meteor.subscribe('teams'), Meteor.subscribe('users'), Meteor.subscribe('survey')];
+  },
+
+  action() {
+    Session.set('title', 'SEMS | Admin');
+    this.render('surveyAdmin');
+  },
+
+});
+
+Router.route('/survey/admin/:slug/', {
+  name: 'admin.teamInfo',
+  layoutTemplate: 'teamInfoAdmin',
+
+  waitOn() {
+    return [Meteor.subscribe('teams'), Meteor.subscribe('fullUsers')];
```

Comment:
why ? shouldn't subscribing to `users` as admin return this ? like in `manageUsers`